### ISSUE TITLE
Revise tab order

### ISSUE THREAD

**Issue by njisaf** [njisaf/AvantVTT#16]
The sheet's tab order could better reflect the order of significance and frequency of access. If the combat tab is consolidated into the gear tab, the tab order should be the following: Core, Gear, Actions, Talents & Powers, Skills, Features. If the combat tab is consolidated in the action tab, the tab order should be the following: Core, Actions, Talents & Powers, Skills, Gear, Features. My current personal preference is the latter.

**Comment by njisaf** [comment]
Wait on this until we confirm Combat tab changes.
